name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages.properties

```diff
@@ -1216,7 +1224,7 @@ LabelDoImport = Import
 
 LabelDoNotImport = Do not import
 
-LabelDuplicateSecurity = Duplicate security
+LabelDuplicateSecurity= Duplicate security
```

Comment:
Please use [ResourceBundle Editor](https://github.com/buchen/portfolio/blob/master/CONTRIBUTING.md#install-eclipse-plugins) when creating, editing and translating labels.